client/src/containers/map/markers/home-markers/index.tsx

```diff
@@ -37,9 +37,11 @@ const StoryMarkers = () => {
             geometry: { type: 'Point', coordinates: [attributes?.longitude, attributes?.latitude] },
             properties: {
               category: attributes?.category?.data?.attributes?.slug,
```

Comment:
This should be changed to a general category but i am not sure where its been used

Reply:
me neither... we could add a TODO comment in the code and they can have a look once they come back